tests/core/reader/test_SensorReader.py

```diff
@@ -1,10 +1,19 @@
 import pytest
+from _pytest.logging import LogCaptureFixture
```

Comment:
Please do not import private modules. Use `pytest.LogCaptureFixture` instread.

Reply:
I get an error if I do this:

```
E   ModuleNotFoundError: No module named 'pytest.logging'
```

I'm just following loguru's own docs here: https://loguru.readthedocs.io/en/stable/resources/migration.html#making-things-work-with-pytest-and-caplog